Gemfile.lock

```diff
@@ -1,45 +0,0 @@
-GEM
```

Comment:
this confuses gem/bundler sometimes. The file implies that the gems are already available, but thats not the case if I clone the repo fresh. We dropped them on all Vox Pupuli repos.